src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,216 @@
+        String insertDriversQuery =
+                "INSERT INTO cars_drivers (car_id, driver_id) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                    PreparedStatement statement
+                            = connection.prepareStatement(
+                                    insertDriversQuery)) {
+            statement.setLong(1, car.getId());
+            for (Driver driver : car.getDrivers()) {
+                statement.setLong(2, driver.getId());
+                statement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert drivers to car " + car, e);
+        }
+        return car;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE cars SET is_deleted = TRUE WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                    PreparedStatement deleteDriverStatement
+                                = connection.prepareStatement(query)) {
+            deleteDriverStatement.setLong(1, id);
+            return deleteDriverStatement.executeUpdate() > 0;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't delete car with id " + id, throwable);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String selectDrivers = "SELECT c.id, c.title, c.manufacturer_id"
+                + " FROM cars c JOIN cars_drivers cd ON "
+                + "cd.car_id = c.id"
+                + " WHERE cd.car_id = ? AND c.is_deleted = false";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                    PreparedStatement selectStatement
+                        = connection.prepareStatement(selectDrivers)) {
+            selectStatement.setLong(1, driverId);
+            ResultSet resultSet = selectStatement.executeQuery();
+            Car car = new Car();
+            while (resultSet.next()) {
+                Long id = resultSet.getObject("id", Long.class);
+                car.setDrivers(getDriversById(id));
+                car.setId(id);
+                car.setManufacturer(getManufacturerBySet(resultSet));
+                car.setTitle(resultSet.getString(2));
+                cars.add(car);
+            }
+            return cars;
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Couldn't get driver by set.",
+                    throwables);
+        }
+    }
+
+    private Manufacturer getManufacturerBySet(ResultSet resultSet) {
+        Manufacturer manufacturer = null;
+        try {
+            manufacturer = new Manufacturer(
+                    resultSet.getString(6),
+                    resultSet.getString(7)
+            );
+            manufacturer.setId(resultSet.getObject(5, Long.class));
```

Comment:
Pretty sure its easier to get that by names rather than by columns.
1. For reading - I do not get what those magic numbers mean
2. For your understanding and economy of your brain resourses, so that you don't have to use them to memorize columns that you are aiming for here
3. For code extension - what if you will have to use it again, but your data will be having another look from what you are using right now. Or another porgrammer will use your method, and he won't know what's inside of it - his code will break